language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>Again here it might be better to show the <em>before</em> vs the <em>after. You can even do a count of the character or word length before/after to show the % reduction via summarization and the final word count.</em></p><p> </p><p><em>The LLM may not always return a summary within 25 words or less, so it may also be useful to warn the users that depending on the needs of the user, this may require some more prompt engineering.</em>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Not a must-have, but just wanted to seek any final thoughts on this before resolving this comment -- cheers!</p>

Reply:
<p>heya, Alicia added the before/after and word count difference, we also added this disclaimer after the code block: <em>The word count of the results may not always be within the the 25 words requested and so further&nbsp;</em><a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/language/intro_prompt_design.ipynb" target="_blank"><em>prompt engineering</em></a><em>&nbsp;may be required.  </em>Shout if we've missed anything</p><p> </p>